integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -638,44 +647,181 @@ void testCreateDomainNoFilesAtSourceModelHome() {
+    // create domain and verify it is failed
+    logger.info("Creating domain custom resource for domainUid {0} in namespace {1}",
+            domainUid, domainNamespace);
+    assertDoesNotThrow(() -> createDomainCustomResource(domainCR),
+            String.format("Create domain custom resource failed with ApiException for %s in namespace %s",
+                    domainUid, domainNamespace));
+    // check the introspector pod log contains the expected error message
+    String expectedErrorMsg = "createDomain failed to find a model file in archive";
+
+    // check the domain event contains the expected error message
+    checkDomainEventContainsExpectedMsg(opNamespace, domainNamespace, domainUid, DOMAIN_FAILED,
+            "Warning", timestamp, expectedErrorMsg);
+
+    // check the operator pod log contains the expected error message
+    checkPodLogContainsString(opNamespace, operatorPodName, expectedErrorMsg);
+
+    // check there are no admin server and managed server pods and services not created
+    checkPodDoesNotExist(adminServerPodName, domainUid, domainNamespace);
+    checkServiceDoesNotExist(adminServerPodName, domainNamespace);
+    for (int i = 1; i <= replicaCount; i++) {
+      checkPodDoesNotExist(managedServerPrefix + i, domainUid, domainNamespace);
+      checkServiceDoesNotExist(managedServerPrefix + i, domainNamespace);
+    }
+
+    // delete domain3
+    deleteDomainResource(domainNamespace, domainUid);
+    String configMapName1 = "modelfiles1-cm";;
+
+    //add model files to configmap and verify domain is running
+    cmFiles.add(
+            Paths.get(MODEL_DIR + "/" + MII_BASIC_WDT_MODEL_FILE));
+    cmFiles.add(Paths.get(MODEL_DIR + "/multi-model-one-ds.20.yaml"));
+    assertDoesNotThrow(
+            () -> createConfigMapForDomainCreation(
+                    configMapName1, cmFiles, domainNamespace, this.getClass().getSimpleName()),
+            "Create configmap for domain creation failed");
+
+    // create domain custom resource using auxiliary image
+    logger.info("Creating domain custom resource with domainUid {0} and auxiliary image {1}",
+            domainUid, miiAuxiliaryImage12);
+    Domain domainCR1 = createDomainResourceWithAuxiliaryImage40(domainUid, domainNamespace,
+            WEBLOGIC_IMAGE_TO_USE_IN_SPEC, adminSecretName, OCIR_SECRET_NAME,
+            encryptionSecretName, replicaCount, List.of("cluster-1"), auxiliaryImagePathCustom,
+            miiAuxiliaryImage12);
+    assertNotNull(domainCR1, "failed to create domain resource");
+    domainCR1.spec().configuration().model().configMap(configMapName1);
+    createDomainAndVerify(domainUid, domainCR1, domainNamespace,
+            adminServerPodName, managedServerPrefix, replicaCount);
+  }
+
+  /**
+   * Create a domain using auxiliary image with custom wdtModelHome and wdtInstallHome
+   * where the wdtModelHome is placed
+   * under the wdtInstallHome directory, or same directory. Ditto for vice-versa.
```

Comment:
Describe the usecase clearly with out using the word 'ditto'  Give the example of the the parameter with the actual value used in the usecase

Reply:
done